.eslintrc

```diff
@@ -4,18 +4,16 @@
 		"es2020": true,
 		"jest": true
 	},
-	"extends": ["plugin:jsdoc/recommended", "plugin:prettier/recommended"],
```

Comment:
As we're using typescript, I'm OK removing JSDoc in theory (as TS will cover most bits), but I'd rather keep some form of required function description linting.

Reply:
I'm fine with reverting this removal.